controllers/test.go

```diff
@@ -0,0 +1,26 @@
+// TODO: remove this temporary controller later
```

Comment:
How about renaming this controller dashboard or home so that it can be kept for later?